src/components/auth/ForgotPassword.jsx

```diff
@@ -0,0 +1,101 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+import FormInput from "../common/FormInput";
+import { useNavigate } from "react-router-dom";
+import Modal from 'react-modal';
+
+Modal.setAppElement('#root'); // Set the app element for accessibility
+
+const ForgotPassword = () => {
+  const [email, setEmail] = useState('');
+  const [message, setMessage] = useState('');
+  const [formErrors, setFormErrors] = useState({});
+  const [isModalOpen, setIsModalOpen] = useState(false);
+  const navigate = useNavigate();
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+
+    if (!email) {
+      setFormErrors({
+        ...formErrors,
+        email: "Email is required",
+      });
+      return;
+    }
+
+    try {
+      const response = await axios.post('http://localhost:8000/api/v1/forgot-password', {
+        email: email
+      }, {
+        headers: {
+          'Content-Type': 'application/json'
+        }
+      });
+      setMessage(''); // Clear the message
+      setIsModalOpen(true); // Open the modal
+    } catch (error) {
+      console.error('Error:', error);
+      if (error.response && error.response.status === 401) {
+        setMessage('Invalid credentials');
+      } else {
+        setMessage(error.response?.data?.message || 'Wrong credentials');
+      }
+    }
+  };
+
+  return (
+    <div className="flex items-center justify-center min-h-screen">
+      <div className="w-full max-w-md p-8 space-y-6 rounded shadow-md">
```

Comment:
without shadow-md the form looks better

Reply:
I like how it looks with shadow, it takes a sense  of completion. But I'm open to ideas.

Reply:
About shadow. It is better to have the consistent theming throughout the app. Maybe it worth to discuss on the team meeting: does the forms need shadow or not? Because current mock-up does not have shadow with forms

Reply:
Yes, now I see we have a lot of forms and all of them looks better with no shadow, just wanted to try out the idea. Fix it